src/dependency-manager/manager.ts

```diff
@@ -462,7 +474,8 @@ export default abstract class DependencyManager {
       const component_config = transformComponentSpec(component_spec);
 
       if (interpolate && validate) {
-        this.validateRequiredParameters(component_config, context.parameters || {});
+        this.validateRequiredSecrets(component_config, context.secrets || {});
+        this.validateRequiredSecrets(component_config, context.parameters || {}); // TODO: 404: remove
```

Comment:
Does this need to be removed?

Reply:
I see you changed it to secretes, but not sure if you meant to remove the parameter part now

Reply:
I was just leaving this for now, to be removed later. The validation function hasn't functionally changed